src/controllers/map-controls.js

```diff
@@ -59,9 +59,9 @@ const propTypes = {
 
   constraints: PropTypes.object,
 
-  perspectiveEnabled: PropTypes.bool,  // Enables perspective control event handling
+  dragToRotate: PropTypes.bool,  // Enables perspective control event handling
```

Comment:
> Why do we have a copy of react-map-gl components in deck.gl?

I used to do integration experiments with non-mapbox maps, and I was also investigating building on a family of controllers that would follow similar principles.

It is another chapter in the never ending story of what should reasonably be common components between these two frameworks, but with no good place to put them.

We can remove it for now. Another PR though.